test/unit_PolynomialZonotope.jl

```diff
@@ -0,0 +1,17 @@
+N = Float64
+
+c = zeros(N, 2)
+E1, E2 = diagm(N[-1, 0.5]), N[1 1; 0.5 0.3]
+E = [E1, E2];
+F2 = N[-0.5 1]';
+F = [F2];
+G = diagm(N[0.3, 0.3]);
+p = PolynomialZonotope(c, E, F, G)
+
+@test dim(p) == 2
+@test order(p) == 7//2
+@test polynomial_order(p) == 2
+
+# type-specific concrete methods
+scale(N(0.5), p)
+linear_map(N[1.0 2.0; 2.0 5.0], p)
```

Comment:
Add a unit test for `minkowski_sum`?

Reply:
ok. i plan to add proper `@test`'s when with #230 gets in